vsintegration/src/FSharp.Editor/Completion/CompletionProvider.fs

```diff
@@ -218,8 +219,8 @@ type internal FSharpCompletionProvider
         asyncMaybe {
             let document = context.Document
             let! sourceText = context.Document.GetTextAsync(context.CancellationToken)
-            let defines = projectInfoManager.GetCompilationDefinesForEditingDocument(document)
-            do! Option.guard (CompletionUtils.shouldProvideCompletion(document.Id, document.FilePath, defines, sourceText, context.Position))
+            //let defines = projectInfoManager.GetCompilationDefinesForEditingDocument(document)
```

Comment:
Derp, I need to just delete this line; provided my change to calling `ShouldTriggerCompletion` be the right thing